### PR TITLE
fix nomination bug when loading platforms

### PR DESCRIPTION
**Pull request by zkat**
This fixes an issue where VS nominations were nominating the wrong framework.

### CODE REVIEW

File: src/NuGet.Clients/NuGet.SolutionRestoreManager/VSNominationUtilities.cs [hunk 1]
```diff
@@ -155,7 +155,7 @@ internal static RuntimeGraph GetRuntimeGraph(IEnumerable targetFrameworks)
         internal static NuGetFramework GetTargetFramework(IVsProjectProperties properties, string projectFullPath)
         {
             var targetFrameworkMoniker = GetPropertyValueOrNull(properties, ProjectBuildProperties.TargetFrameworkMoniker);
-            var targetPlatformMoniker = GetPropertyValueOrNull(properties, ProjectBuildProperties.TargetPlatformIdentifier);
+            var targetPlatformMoniker = GetPropertyValueOrNull(properties, ProjectBuildProperties.TargetPlatformMoniker);
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.SolutionRestoreManager/VSNominationUtilities.cs:158]: Can you pelase add a test. 

We didn't have any tests for these scenarios and that's why the bug was introduced. 

Let's cover that gap: https://github.com/NuGet/NuGet.Client/blob/e97d556a83750bb4f8017e3640bc699d04832657/test/NuGet.Clients.Tests/NuGet.SolutionRestoreManager.Test/VsSolutionRestoreServiceTests.cs#L73

You can use that as a template...just provide a net50-win123 framework.

**Comment by kartheekp-ms** [comment]
This PR is missing a tracking issue [here](https://github.com/nuget/home/issues)

**Comment by zkat** [comment]
@kartheekp-ms this is related to an AzDO issue. I'm not sure if we're supposed to mirror similar issues on our end?

**Comment by kartheekp-ms** [comment]
> @kartheekp-ms this is related to an AzDO issue. I'm not sure if we're supposed to mirror similar issues on our end.?

I remember creating GitHub issues in the past for UI delays logged in AzDO. For e.g. NuGet/Home#8736

**Review by nkolev92 — APPROVED** [review]

**Review by aortiz-msft — APPROVED** [review]
:shipit:

**Pull request merged**